cmd/api/api_server.go

```diff
@@ -22,9 +24,25 @@ var (
 	rdb      *redis.Client
 	ctx      context.Context
 	redisKey = "Stock"
+	logger   *zap.Logger
 )
 
+// ClusterListResponse models the Cluster list response returned by this API
```

Comment:
WDYT about
`// ClusterListResponse represents the API response containing a list of clusters` ?